docs/tutorials/basics/60_methods.md

```diff
@@ -107,3 +107,60 @@ say_hello 3
 
 Overloading isn't defined just by type restrictions. The number of arguments as well as named arguments are also
 relevant characteristics.
+
+## Return value
+
+Methods may also return a value:
+
+```crystal-play
+def adds_2(n : Int32 = 0)
+  n + 2
+end
+
+puts adds_2 40
+```
+
+The keyword `return` is not necessary and methods will always return the result of executing the last line.
```

Comment:
```suggestion
Note how there's no `return` keyword: methods always return the result of executing the last line.
```

Reply:
I preferred the previous wording. Or at least the suggested wording can be misleading because people may see it as just "there's no `return` keyword" (whatsoever in the language)

Reply:
That's right! I didn't want the reader to misread it as "the language don't have `return` keyword". That's why I went with _The keyword `return` is not necessary_

Reply:
I add a "Note block" to highlight the use of `return`.

Regarding the content maybe we can say something like:
_In the previous example, the use of the keyword `return` is redundant since methods always return the evaluation of the last line._